docs/releases/1.9.txt

```diff
@@ -195,6 +195,11 @@ Database backend API
   doesn't implement this. You may want to review the implementation on the
   backends that Django includes for reference (:ticket:`24245`).
 
+Usernames are now up to 254 characters long
```

Comment:
This needs more details: `AbstractUser.username` ....
What are the implications for custom user models that inherit from `AbstractUser`? I assume a migration will be needed.